### PR TITLE
Fix Layout/IndentArray & Layout/IndentHash

### PR DESCRIPTION
**Pull request by jaysonesmith**
## Details

- Spacing fun!

## Motivation and Context

Working to help solve issue [1021](https://github.com/cucumber/cucumber-ruby/issues/1021)!

## How Has This Been Tested?

`bundle exec rake` :+1:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Rubocop style fixes

### CODE REVIEW

File: features/lib/support/fake_wire_server.rb [hunk 1]
```diff
@@ -67,13 +67,13 @@ def handle(data)
...
-                      :exception => e.class
-                    }
-                  ].to_json
+        'fail',
+        {
+          :message => e.message,
+          :backtrace => e.backtrace,
+          :exception => e.class
+        }
+      ].to_json
```
Comment (olleolleolle) [inline features/lib/support/fake_wire_server.rb]: I'm curious: Do we support Ruby < 1.9? Can we used `{ message: e.message }` colon syntax in Hashes? Are we already doing that?

Comment (jaysonesmith) [reply features/lib/support/fake_wire_server.rb]: The lowest we support is 2.2 now. I can try changing it. :)

Comment (jaysonesmith) [reply features/lib/support/fake_wire_server.rb]: @olleolleolle Hash syntax is handled by the Style/HashSyntax cop, but I'll change this one now. :)

**Comment by jaysonesmith** [comment]
@olleolleolle all set on this one!

**Pull request merged**